Revamped Gateway on Kubernetes install guide (#6695)
Kong/docs.konghq.com: app/_src/gateway/install/kubernetes/proxy.md

---
title: Install Kong Gateway
book: kubernetes-install
chapter: 3
---

This guide shows how to deploy {{ site.base_gateway }} on Kubernetes without [{{ site.konnect_saas }}](/konnect/gateway-manager/data-plane-nodes/) or [{{ site.kic_product_name }}](/kubernetes-ingress-controller/latest/get-started/).

{:.important}
> **{{ site.konnect_saas }} is recommended for new installations to reduce deployment complexity.**
> <br />
> Let Kong run the control plane and database for you. With {{ site.konnect_saas }}, you only need to run the data planes. <a href="https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=kubernetes-install">Get started in under 5 minutes</a>.

> These instructions configure {{ site.base_gateway }} to use separate containers for each role (e.g. Control Plane, Data Plane, Kong Manager). This is the recommended production installation method.

Author: ```suggestion
These instructions configure {{ site.base_gateway }} to use separate control plane and data plane deployments. This is the recommended production installation method.
```
